composer.json

```diff
@@ -11,6 +11,7 @@
   "require": {
     "php": ">=7.2",
     "ext-json": "*",
+    "ext-ev": "*",
```

Comment:
Instead of require, I would use [suggest](https://getcomposer.org/doc/04-schema.md#suggest). This should be a separate commit.